website/addons/googledrive/utils.py

```diff
@@ -27,6 +28,7 @@ def to_hgrid(item, node, path):
         'kind': 'folder',
         'name': safe_name,
         'addon': 'googledrive',
+        'provider': 'googledrive',  # TODO: remove duplication -- APIv2
```

Comment:
Language usage needs to be normalized -- APIv2 requires the service provider to be keyed to `'provider'`, current front-end / v1 code's key is `'addon'`. Updating this everywhere seemed out of scope.